src/pages/gown.js

```diff
@@ -0,0 +1,375 @@
+    background-color: ${props => props.theme.colors.backgroundGray};
+    width: 50%;
+    padding-left: 10%;
+    padding-top: 5px;
+  }
+`;
+
+const DesktopFlexColumn = styled.div`
+@media (min-width: ${props => props.theme.breakpoints.tablet}) {
+  width: 50%;
+  display: flex;
+  flex-direction: column;
+}
+`
+
+const Name = styled.p`
+  font-size: 22px;
+  font-style: normal;
+  font-weight: bold;
+  margin: 10px 0 10px 8.8%;
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    width: 80%;
+    padding-left: 15%;
+    margin-bottom: 12%;
+    /* text-align: center; */
+  }
+`;
+
+const Brief = styled.p`
+display: none;
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    display: inline-block;
+    width: 54%;
+    margin-left: 8%;
+    margin-bottom: 12%;
+    font-family: Raleway;
+    font-style: 800;
+    font-weight: bold;
+    font-size: 14px;
+  }
+`;
+
+const DetailsFeatures = styled.p`
+display: none;
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    display: inline-block;
+    width: 54%;
+    margin-left: 8%;
+    margin-bottom: 4%;
+    font-family: Raleway;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 14px;
+  }
+`;
+
+const Description = styled.p`
+display: none;
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    display: inline-block;
+    width: 54%;
+    margin-left: 8%;
+    font-family: Raleway;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 14px;
+  }
+`;
+
+const DetailsAndFeaturesContainer = styled.div`
+  display: flex;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 20px;
+  margin: 0 0 36px 8.8%;
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    border: 1px solid red;
+    width: 40%;
+    display: none;
+  }
+`;
+
+const Details = styled.p`
+@media (min-width: ${props => props.theme.breakpoints.tablet}) {
+}`;
+
+const DesktopButtonContainer = styled.p`
+  display: none;
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+    display: flex;
+    margin-left: 8%;
+    margin-top: 24%;
+}
+`;
+
+const MobileButtonContainer = styled.p`
+  display: flex;
+  justify-content: center;
+  border-bottom: 48px solid ${props => props.theme.colors.primaryPink};
+  @media (min-width: ${props => props.theme.breakpoints.tablet}) {
+  display: none;
+}
+`;
+
+const DifferentAnglesContainer = styled.div`
+display: none;
+@media (min-width: ${props => props.theme.breakpoints.tablet}) {
+  display: flex;
+  margin-top: 30px;
+  margin-left: 10%;
+  border-bottom: 200px solid ${props => props.theme.colors.backgroundGray};
+}
+`
+
+const DifferentAngles = styled.img`
+display: none;
+@media (min-width: ${props => props.theme.breakpoints.tablet}) {
+  display: inline-block;
+  width: 24%;
+  height: 300px;
+  margin: 0 26px 0 0;
+}
+`
+
+const GownsPage = () => (
+  <Layout>
+    <NavBar />
+    <LogoContainer>
+      <Logo notag />
```

Comment:
The logo doesn't seem to be appearing in the browser.

Reply:
Logo issue is a merge issue I believe